frontend/src/framework/theme/styles/components.styles.ts

```diff
@@ -73,6 +73,10 @@ const components = {
                     },
                 },
             },
+            gray: {
```

Comment:
Maybe call it a bit different like a player or something? Because others are named not by a color used.

Reply:
But this style can be used not only in a player... Do you have better suggestions for the name?

Reply:
Maybe round?

Reply:
But it is not about shape here. I only applied colors